tools/clang/unittests/HLSL/PixTest.cpp

```diff
@@ -4187,20 +4196,20 @@ void main()
 
       auto Testables = TestStructAnnotationCase(hlsl, optimization);
       if (!choice.IsOptimized) {
-          VERIFY_ARE_EQUAL(1, Testables.OffsetAndSizes.size());
-          VERIFY_ARE_EQUAL(6, Testables.OffsetAndSizes[0].countOfMembers);
-          VERIFY_ARE_EQUAL(0, Testables.OffsetAndSizes[0].offset);
-          VERIFY_ARE_EQUAL(2 * 3 * 32, Testables.OffsetAndSizes[0].size);
+          VERIFY_ARE_EQUAL(1u, Testables.OffsetAndSizes.size());
+          VERIFY_ARE_EQUAL(6u, Testables.OffsetAndSizes[0].countOfMembers);
+          VERIFY_ARE_EQUAL(0u, Testables.OffsetAndSizes[0].offset);
+          VERIFY_ARE_EQUAL(2 * 3 * 32, (int)Testables.OffsetAndSizes[0].size);
```

Comment:
I have a mild preference for making the literals unsigned rather than signing the variable. It's unlikely that it will make a practical difference, but it's consistent with the other locations and technically more correct.